apps/mobile/src/features/psbt-signer/psbt-signer.tsx

```diff
@@ -8,7 +8,10 @@ import { BitcoinOutcome } from '@/features/approver/components/bitcoin-outcome';
 import { BitcoinFeeCard } from '@/features/approver/components/fees/bitcoin-fee-card';
 import { InputsAndOutputsCard } from '@/features/approver/components/inputs-outputs-card';
 import { OutcomeAddressesCard } from '@/features/approver/components/outcome-addresses-card';
+import { useAverageBitcoinFeeRates } from '@/queries/fees/fee-estimates.hooks';
```

Comment:
Would we eventually want a package that exports hooks so that each app _can_, if it desires, use that as a dependency?